src/mds/MDSRank.cc

```diff
@@ -645,9 +645,9 @@ void MDSRank::update_targets()
 
 void MDSRank::hit_export_target(mds_rank_t rank, double amount)
 {
-  double rate = g_conf()->mds_bal_target_decay;
+  double rate = g_conf().get_val<double>("mds_bal_target_decay");
   if (amount < 0.0) {
-    amount = 100.0/g_conf()->mds_bal_target_decay; /* a good default for "i am trying to keep this export_target active" */
```

Comment:
@batrick I noticed the variable rate wasn't reused here but mds_bal_target_decay was re-fetched in the original code. Is there is a reason for this and so should I leave this untouched?

Reply:
I think your fix is appropriate.